index.html

```diff
@@ -275,46 +321,48 @@ <h4 class="caption">Languages</h4>
           </li>
         </ul>
 
-        <ul class="tabs-bottom-line">
-          <li class="tabs-one">
+        <ul class="tools tabs-bottom-line">
+          <li class="tabs">
             <h4 class="caption">Frameworks</h4>
-            <a href="#" class="tag">
+            <a href="#">
               <img
-                class="left-sidebar-two"
+                class="left-sidebar"
                 alt="open-tab"
                 src="arrow2.png"
               />
             </a>
           </li>
         </ul>
 
-        <ul class="tabs-bottom-line">
-          <li class="tabs-two">
+        <ul class="tools tabs-bottom-line">
+          <li class="tabs">
             <h4 class="caption">Skills</h4>
-            <a href="#" class="tag">
+            <a href="#">
               <img
-                class="left-sidebar-three"
+                class="left-sidebar"
                 alt="open-tab"
                 src="arrow2.png"
               />
             </a>
+
+            
           </li>
         </ul>
       </div>
     </section>
-
-
-    <section class="contact-form" >
-
-      <div class="contact-me"> 
-        <h2 class="contact">Contact me</h2> 
-        <p class="contact-paragraph"> If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
-
-        </p> 
-      </div> 
+    
+    <section class="contact-form" id="contact">
```

Comment:
[OPTIONAL] It would look a lot nice if you reduce the width of the contact form to match the Figma design 🤗

Your design:
![image](https://user-images.githubusercontent.com/73714615/186907011-e2948558-fe5b-4372-860d-45a912aecd59.png)

Figma design:
![image](https://user-images.githubusercontent.com/73714615/186907049-9a564e79-4d04-4cce-8586-5369ad27d10f.png)